src/UI/Internal/Text.elm

```diff
@@ -11,7 +11,7 @@ import UI.Utils.Element as Element
 
 
 type Text
-    = Text (List Span)
+    = Text (List Span) CombinedOptions
```

Comment:
What do you think about having `TextOptions` and `SpanOptions`?

Reply:
I like it!